### PR TITLE
Feature/AP-4510 Updating Grafana Docker image base version

### PR DESCRIPTION
**Pull request by mishah334**
## Which issue this PR fixes

- Updating `grafana` docker image base version to match with `astronomer`. 

## Summary of changes

- Updating `grafana` docker image.
- Updating `version.txt`.

## Images are updated by this PR

- [ ] alertmanager
- [ ] auth-sidecar
- [ ] awsesproxy
- [ ] blackbox-exporter
- [ ] configmap-reloader
- [ ] curator
- [ ] dind-golang
- [ ] elasticsearch
- [ ] elasticsearch-exporter
- [ ] fluentd
- [ ] git-daemon
- [ ] git-sync
- [x] grafana
- [ ] keda
- [ ] keda-metrics-apiserver
- [ ] kibana
- [ ] kube-state
- [ ] kubed
- [ ] nats-exporter
- [ ] nats-server
- [ ] nats-streaming
- [ ] nginx
- [ ] nginx-es
- [ ] node-exporter
- [ ] openresty
- [ ] pgbouncer
- [ ] pgbouncer-exporter
- [ ] postgres-exporter
- [ ] postgresql
- [ ] prometheus
- [ ] redis
- [ ] registry
- [ ] statsd-exporter
- [ ] vector

## Checklist

Place an '[x]' (no spaces) in all applicable fields. Please remove unrelated fields.

- [x] version.txt is updated in the changed image(s)

If a security scan fails for an unchanged image...

- [ ] a fix has been applied OR...
- [ ] an [issue]() has been created

If adding a new image:

- [ ] the directory has the same name you intend it to be published as, less a preceding "ap-"
- [ ] the directory includes a file "version.txt", with version matching the underlying software version
- [ ] the file .github/PULL_REQUEST_TEMPLATE.md is updated to include the image in the checklist
- [ ] execute the script .circleci/generate_circleci_config.py, commit changes to .circleci/config.yml

### CODE REVIEW

**Review by andriisoldatenko — APPROVED** [review]

File: grafana/version.txt [hunk 1]
```diff
@@ -1 +1 @@
-8.3.7-2
+8.4.5-1
```
Comment (danielhoherd) [inline grafana/version.txt:1]: We should make this 8.4.5-2 since there is already a 8.4.5-1 tag.

File: grafana/version.txt [hunk 2]
```diff
@@ -1 +1 @@
-8.3.7-2
+8.4.5-2
```
Comment (mishah334) [inline grafana/version.txt:1]: `8.4.5-1` was released earlier.

File: grafana/version.txt (hunk 1 above)
Comment (mishah334) [reply grafana/version.txt]: Image was backport. Change: https://github.com/astronomer/ap-vendor/commit/c64bb75a1e82d5409e080e920b6458e5c4c9e539

Comment (mishah334) [reply grafana/version.txt]: Done

**Review by danielhoherd — APPROVED** [review]

**Pull request merged**